docs/reference/datatiers.asciidoc

```diff
@@ -1,100 +1,115 @@
-* <<data-hot-node, data_hot>>
-* <<data-warm-node, data_warm>>
-* <<data-cold-node, data_cold>>
-
-The more generic <<data-node, data role>> is not a data tier role, but
-it is the default node role if no roles are configured. If a node has the
-<<data-node, data>> role we treat the node as if it has all of the tier
-roles assigned.
+== Data tiers
 
-[[content-tier]]
-==== Content tier
+A _data tier_ is a collection of nodes with the same data role that 
+typically share the same hardware profile: 
 
-The content tier is made of one or more nodes that have the <<data-content-node, data_content>>
-role. A content tier is designed to store and search user created content. Non-timeseries data
-doesn't necessarily follow the hot-warm-cold path. The hardware profiles are quite different to
-the <<hot-tier, hot tier>>. User created content prioritises high CPU to support complex
-queries and aggregations in a timely manner, as opposed to the <<hot-tier, hot tier>> which
-prioritises high IO.
-The content data has very long data retention characteristics and from a resiliency perspective
-the indices in this tier should be configured to use one or more replicas.
+* <<content-tier, Content tier>> nodes handle the indexing and query load for a collection 
+of content such as a product catalog.
+* <<hot-tier, Hot tier>> nodes handle the indexing load for time series data such as logs or metrics
+and hold your most recent, most-frequently-accessed data. 
+* <<warm-tier, Warm tier>> nodes hold time series data that is accessed less-frequently 
+and rarely needs to be updated.
+* <<cold-tier, Cold tier>> nodes hold time series data that is accessed occasionally and never updated.
 
-NOTE: new indices that are not part of <<data-streams, data streams>> will be automatically allocated to the
-<<content-tier>>
+When you index documents directly to a specific index, they remain on content tier nodes indefinitely. 
 
-[[hot-tier]]
-==== Hot tier
+When you index documents to a data stream, they initially reside on hot tier nodes. 
+You can configure <<index-lifecycle-management, {ilm}>> ({ilm-init}) policies
+to automatically transition your time series  data through the hot, warm, and cold tiers 
+according to your performance, resiliency and data retention requirements. 
+
+You assign a node to one or more data tiers by setting its <<data-node, data role>> in `elasticsearch.yml`. 
+For example, you might assign your highest-performance nodes to both the hot and content tiers:
 
-The hot tier is made of one or more nodes that have the <<data-hot-node, data_hot>> role.
-It is the {es} entry point for timeseries data. This tier needs to be fast both for reads
-and writes, requiring more hardware resources such as SSD drives. The hot tier is usually
-hosting the data from recent days. From a resiliency perspective the indices in this
+[source,yaml]
+--------------------------------------------------
+node.roles:
+   - data_hot
+   - data_content
```

Comment:
In general, we stick to single-line arrays in our example `elasticsearch.yml`, so I think I'd prefer:

```suggestion
node.roles: ["data_hot", "data_content"]
```